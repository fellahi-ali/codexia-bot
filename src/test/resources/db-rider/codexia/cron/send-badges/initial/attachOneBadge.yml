codexia_project:
    -
        id: 5
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 123
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

codexia_badge:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_project_id: 5
        badge: "bad"
        deleted_at: null
