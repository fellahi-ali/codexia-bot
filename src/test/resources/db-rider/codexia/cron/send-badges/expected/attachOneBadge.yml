codexia_project:
    -
        external_id: 123
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

codexia_badge:
    -
        codexia_project_id: 5
        badge: "bad"
        deleted_at: null
